File: Problems/35. Search Insert Position/javascript/search_position.js
const searchInsert = (nums, target) => {
    if (nums.includes(target)) {
        // if nums array has target in it. return the index of the target
        return nums.indexOf(target);
    } else {
        // if target is not in nums array push the number into the array
        nums.push(target);
        // sort the array with the target number added
        nums.sort((a,b) => a-b);
        // return the index of the target in the newly sorted array
        return nums.indexOf(target);
    }
}

File: Problems/35. Search Insert Position/python/search_position.py
class Solution:
    def searchInsert(self, nums, target):
        if target in nums:
        # if the target number is in the list of numbers return index where that number is in the nums list
            return nums.index(target)
        else: 
        # if the target number is not in the list add that number to the list.
            nums.append(target)
        return sorted(nums).index(target)
        # sort the list with the added target number and find its index in the sorted list.